oak_containers_orchestrator/src/main.rs

```diff
@@ -58,8 +59,21 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .await
         .map_err(|error| anyhow!("couldn't get application config: {:?}", error))?;
 
-    let attestation_report_generator = Arc::new(EmptyAttestationReportGenerator);
+    let dice_builder = oak_containers_orchestrator::dice::load_stage1_dice_data()?;
+    let additional_claims = oak_containers_orchestrator::dice::measure_container_and_config(
+        &container_bundle,
+        &application_config,
+    );
     let encryption_key_provider = Arc::new(EncryptionKeyProvider::new());
+    // Ignore the signing key for now.
+    let (_signing_key, verifying_key) = generate_ecdsa_key_pair();
+
+    let dice_evidence = dice_builder.add_application_keys(
+        additional_claims,
+        &encryption_key_provider.get_serialized_public_key(),
+        &verifying_key,
+    )?;
+    let attestation_report_generator = Arc::new(EmptyAttestationReportGenerator);
```

Comment:
Could you please add a TODO(https://github.com/project-oak/oak/issues/4074) that this is to be removed once DICE attestation is implemented?

Reply:
Maybe in the next PR to not rerun the CI

Reply:
Done.